packs/spell-effects/spell-effect-dutiful-challenge.json

```
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Dutiful Challenge",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Dutiful Challenge]</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Dutiful Challenge]</p>\n<p>The target takes a -1 status penalty to attack rolls, damage rolls, and skill checks against creatures other than you, and you take a -1 status penalty to attack rolls, damage rolls, and skill checks against creatures other than the target.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Dutiful Challenge]</p>\n<p>You take a -1 status penalty to attack rolls, damage rolls, and skill checks against creatures other than the marked creature.</p>"
```
I think this is clearer about what the effect does